stock_picking_on_hold/models/sale_order.py

```diff
@@ -0,0 +1,21 @@
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    payment_method_id = fields.Many2one(
```

Comment:
IMHO, this has to go in another module

Reply:
In sale-workflow should be good.

Reply:
I've created PR in sale-workflow you can refer this PR :- https://github.com/OCA/sale-workflow/pull/2073